tools/runners/run-test-suite.sh

```diff
@@ -201,7 +209,11 @@ then
     ENGINE_ARGS="--snapshot $ENGINE_ARGS"
 fi
 
-echo "[summary] ${ENGINE#$ROOT_DIR} $ENGINE_ARGS ${TESTS#$ROOT_DIR}: $passed PASS, $failed FAIL, $TOTAL total, $ratio% success"
+echo -e "\n[summary] ${ENGINE#$ROOT_DIR} $ENGINE_ARGS ${TESTS#$ROOT_DIR}\n"
+echo -e " TOTAL:  $TOTAL"
+echo -e "$TERM_GREEN PASS:   $passed"
```

Comment:
It will not cause problems here but in general it's better to clear the color settings once they are not needed.

Reply:
In the string, is the space between the color code and the PASS intentional? It makes the line indented by one space. If it's only there to separate the variable expansion from the PASS string then it can be written as `"${TERM_GREEN}PASS`.